### DISCUSSION TITLE
Mobile UI Issue With Custom Icon in 1.7.4

### DISCUSSION THREAD

**Discussion by MazzicCron** [Akifyss/obsidian-border#355] [Q&A]
Hi folks...I just updated Obsidian on my Android phone yesterday and started seeing the issue in the screenshot here. I updated them theme and all my plugins, and tried disabling a bunch of the other community plugins, but it persists. Anyone ever seen anything like this or know how to fix it? 

I've tried...
-Restarting the phone
-Restarting the app
-Turning off community plugins 
-Turning off the vivid Bold Folder Color option 

![Screenshot_20241018-004426](https://github.com/user-attachments/assets/9b53126c-bc91-4292-a944-2a86678937af)

**Comment by Akifyss** [comment]
Sorry, I can't replicate the issue. Could you try it in a new vault?
![1729256124988C3C341E25089ED91](https://github.com/user-attachments/assets/4f399e0e-bdbf-477c-9034-27a3139822dc)

**Reply by MazzicCron** [reply]
Hello! When I create a totally new Vault and install the theme, it doesn't give the weird double icon.

**Reply by MazzicCron** [reply]
I have also tried uninstalling and re-installing the app, and for this specific Vault, it re-occurs post re-install.
